crypto-ffi/bindings/swift/WireCoreCrypto/WireCoreCrypto/CoreCrypto.swift

```diff
@@ -90,10 +90,12 @@ public protocol CoreCryptoProtocol {
 public final class CoreCrypto: CoreCryptoProtocol {
 
     let coreCrypto: WireCoreCryptoUniffi.CoreCrypto
+    let keystoreURL: URL?
 
     /// Initialize CoreCrypto with a Ffi instance
-    private init(_ coreCrypto: WireCoreCryptoUniffi.CoreCrypto) {
+    private init(_ coreCrypto: WireCoreCryptoUniffi.CoreCrypto, keystoreURL: URL? = nil) {
```

Comment:
Why is the type here `URL`? Shouldn't this be a regular, local filesystem path?

Reply:
Right, but I was wondering if we could use a narrower type, e.g. `FilePath` (https://developer.apple.com/documentation/system/filepath), because we don't really need full URL capabilities here.